nextcord/embeds.py

```diff
@@ -456,7 +546,12 @@ def thumbnail(self) -> _EmbedMediaProxy:
         """
         return EmbedProxy(getattr(self, "_thumbnail", {}))  # type: ignore
 
-    def set_thumbnail(self: E, url: MaybeEmpty[Any]) -> E:
+    def set_thumbnail(
+        self: E,
+        url: MaybeEmpty[Any] = EmptyEmbed,
+        *,
+        file: Optional[File] = None,
```

Comment:
I want to say these should both be keyword arguments.

It's a tough call since that will be a breaking change, as it would mean you would no longer be able to make the url positional.

Reply:
Closing as I haven't seen anything further on this + it maintains backwards compat + the note above

Reply:
this *would* be a breaking change yet i would say this is good